src/Scouting.elm

```diff
@@ -1,65 +1,121 @@
-module Scouting exposing (Msg, Model, init, update, main, view )
+module Scouting exposing (Model, Msg, init, main, update, view)
 
 import Browser
-import Html.Attributes exposing (placeholder, value)
-import Html exposing (input , text)
-import Html.Events exposing (onClick , onInput)
+import Html exposing (input, text)
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import Maybe.Extra exposing (unwrap)
 
 
-inputwriter : String -> (String -> Msg) -> String -> Html.Html Msg
-inputwriter placeholderInput msg model =
-    Html.div [] [
-    input [placeholder placeholderInput, onInput msg, value model] []
-    ,text "\n"  
-    ]
+inputWriter : String -> (String -> Msg) -> String -> Html.Html Msg
+inputWriter placeholderInput msg model =
+    Html.div [ style "margin-bottom" "1%" ]
+        [ input [ placeholder placeholderInput, onInput msg, value model ]
+            []
+        ]
```

Comment:
You can put a `style` attribute on the `input` - you don't really need the `div` here, but it will move the input around.
Check it out, maybe you'll like it.